prime-router/src/main/kotlin/ActionMessages.kt

```diff
@@ -1,13 +1,55 @@
 package gov.cdc.prime.router
 
+/**
+ * Possible error codes when parsing/processing messages. Includes Hl7, CSV, FHIR, et al.
+ */
+enum class ErrorCode {
```

Comment:
Moved the errorCode here from Hl7Serializer.kt. This way the error codes can be accessed by any pipeline/format, not just Hl7Serializer. The idea is that this errorCode enum will hold the errorCodes for everything, not just hl7, for example.